tests/Arch/ModelsTest.php

```diff
@@ -46,7 +46,8 @@
         $instance = $model::factory()->create();
 
         $dates = collect($instance->getAttributes())
-            ->filter(fn ($_, $key) => str_ends_with($key, '_at'));
+            ->filter(fn ($_, $key) => str_ends_with($key, '_at'))
+            ->reject(fn ($_, $key) => in_array($key, ['created_at', 'updated_at']));
```

Comment:
Will be better to check if `timestamps` property is enabled (by default) then we can exclude this fields otherwise it's custom columns and casts

Reply:
let's keep it simple for now!